packages/ui-components/src/components/select-option/select-option.scss

```diff
@@ -28,12 +29,17 @@
 	height: var(--select-option-height);
 	padding: 0 $spacing-4x;
 	display: flex;
-	align-items: var(--select-option-flex-alignment);
```

Comment:
I'm unsure if these CSS properties should be passed as CSS variables. Instead, they could be added by styling a components part. But this is just my opinion.